sources/packages/web/src/services/StudentLoanBalanceService.ts

```diff
@@ -0,0 +1,28 @@
+import ApiClient from "@/services/http/ApiClient";
+import { StudentLoanBalanceAPIOutDTO } from "@/services/http/dto";
+
+/**
+ * Client service layer for student loan balance.
+ */
+export class StudentLoanBalanceService {
+  // Shared Instance
+  private static instance: StudentLoanBalanceService;
+
+  static get shared(): StudentLoanBalanceService {
+    return this.instance || (this.instance = new this());
+  }
+
+  /**
+   * Get the monthly part-time loan balance details of
+   * the given student upto 12 months.
```

Comment:
Isn't it better to put the comment as something like `retrieving the last 12 records` rather than the records for the last 12 months?